python/ql/src/experimental/semmle/python/frameworks/Stdlib.qll

```diff
@@ -9,3 +9,136 @@ private import semmle.python.dataflow.new.TaintTracking
 private import semmle.python.dataflow.new.RemoteFlowSources
 private import experimental.semmle.python.Concepts
 private import semmle.python.ApiGraphs
+
```

Comment:
As per a previous discussion, this needs to be moved into it's own filed called `NoSQL.qll`. Adding the comment here so I don't forget.

Reply:
Resolved.